graviti/manager/commit.py

```diff
@@ -267,12 +280,17 @@ def get(self, revision: Optional[str] = None) -> Commit:
             revision: The information to locate the specific commit, which can be the commit id,
```

Comment:
do we need to add if-else to `_generate commits`

Reply:
If the commit id is None, the revision won't be added to the params of the request.